main.py

```diff
@@ -1,10 +1,66 @@
-import init_django_orm  # noqa: F401
+import init_django_orm  # noqa: F401fr
+import json
 
+from pathlib import Path
+from typing import Any, Dict
 from db.models import Race, Skill, Player, Guild
+from django.core.exceptions import ObjectDoesNotExist
+
+
+def read_json() -> dict:
+    path_to_file = Path(__file__).parent.joinpath("players.json")
+    with open(path_to_file, "r") as file_read_stream:
+        players = json.load(file_read_stream)
+
+    return players
+
+
+def get_or_create_race(data: Dict[str, Any]) -> Race:
+    try:
+        race = Race.objects.get(name=data.get("name"))
```

Comment:
I meant to use get_or_create method from Django https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create